tests/rules/date_comparison.py

```diff
@@ -0,0 +1,4 @@
+#OK
+day1 = "3000-01-15"
+#ERROR: match
+day2 = "1906-12-09"
```

Comment:
Could we add a test demonstrating when the string does not resemble a date?

Reply:
Added!